docs/README.md

```diff
@@ -25,8 +25,9 @@
   * [Migrating to Redux](/docs/recipes/MigratingToRedux.md)
   * [Reducing Boilerplate](/docs/recipes/ReducingBoilerplate.md)
   * [Server Rendering](/docs/recipes/ServerRendering.md)
-  * [Computing Derived Data](/docs/recipes/ComputingDerivedData.md)
```

Comment:
I moved this below `Writing Tests` so it has the same order as in `docs/recipes/README.md`